packages/swapper/src/thorchain-utils/memo/assertAndProcessMemo.ts

```diff
@@ -62,18 +62,18 @@ function assertMemoHasAction(action: string | undefined, memo: string): asserts
 const assertIsValidLimit = (limit: string | undefined, memo: string) => {
   assertMemoHasLimit(limit, memo)
 
-  const maybeStreamingSwap = limit.match(/\//g)
-  const [lim, interval, quantity] = limit.split('/')
+  const parts = limit.split('/')
```

Comment:
q: shouldn't this be `streamingParts` ?

Reply:
it would be limit or streaming parts technically. parts.length === 1 implies just limit (standard swap), parts.length === 3 would be streaming swap. I just went with general naming here, we are splitting on `/` and getting back some number of "parts"